lib/tom_queue/deferred_work_set.rb

```diff
@@ -8,6 +8,8 @@ module TomQueue
   #
   class DeferredWorkSet
```

Comment:
I didn't change any mutex and condvar related logic in this class. Can it be removed?